player/src/main/java/com/focusit/jsflight/player/handler/BaseEventHandler.java

```diff
@@ -0,0 +1,30 @@
+package com.focusit.jsflight.player.handler;
+
+import com.focusit.jsflight.script.player.PlayerContext;
+import org.json.JSONObject;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Gallyam Biktashev on 16.02.17.
+ */
+public abstract class BaseEventHandler
+{
+    protected PlayerContext context;
```

Comment:
Having no final protected fields without getters/setter is, as for me, not good, because you coupling hard any decendant class with this one